third_party/libunwindstack/PeCoffEpilog.cpp

```diff
@@ -108,6 +105,17 @@ bool PeCoffEpilogImpl::Init() {
   return true;
 }
 
+bool PeCoffEpilogImpl::MapFromRvaToFileOffset(uint64_t rva, uint64_t* file_offset) {
+  for (const Section& section : sections_) {
```

Comment:
I mentioned this before, but since it's a convenient place to add some details: MapFromRvaToFileOffset was a big reason why unwinding was slow when I profiled the code earlier. I was thinking earlier to just cache the indices of sections that were identified as executable sections (simply by virtue of a PC landing in them), but it's probably cleaner to use the section flags to check if the section is executable as you did to identify the text section in PeCoffInterface.cpp (and will uncover errors when we have a wrong PC). 

(No change needed on the current PR.)

Reply:
Good point. Some sections that are executable (as indicated by flag) might never be executed.